.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 .netlify
 dist/
+package-lock.json
+package.json
```

Comment:
These package files are almost always required when you're working on a project, especially in a team with other developers. Don't add them to `.gitignore`.

Reply:
OK. I understand. In the first homework I have added those two to git-ignore later I corrected but I don't understand why it is still added to git-ignore.